resources/css/style.css

```diff
@@ -590,4 +602,67 @@ video {
         max-width: 1170px;
         /* margin: 5rem auto; */
     }
+
+}
+
+@media only screen and (max-width: 414px){
+
+    .footer-buttons {
+      width: 100%; //override the web width attribute
+    }
+
+    .float-right, .col-block{
+      float:none;
+    }
+
+    .fcontainer {
+      padding: 0%;
+    }
+
+    .footer-buttons .footer-buttons-wrapper {
+      margin: 0 40px 0 40px;
+    }
+
+    .footer-buttons .button {
+      box-sizing: content-box;
+      display: block;
+      float: none;
+      width: auto;
+    }
+
```

Comment:
Can you add the `text-decoration: none;` style to your "Get a Quote", "Contact Us" and "1for1 Match" button anchor elements?

Reply:
Included text-decoration for anchor elements.